frontend/src/components/utils/ServiceStatus.tsx

```diff
@@ -0,0 +1,233 @@
+import React from "react";
+import { OverlayTrigger, Tooltip } from "react-bootstrap";
+import toast from "react-hot-toast";
+import { usePrevious } from "../../hooks/use-previous";
+import {
+  UseSubscriptionResult,
+  useSubscription,
+} from "../../hooks/use-subscription";
+import { styled } from "../../libraries/styles";
+import { MINUTE, SECOND } from "../../libraries/time";
+import { HealthStatus } from "../../types/common";
+import Timer from "./Timer";
+
+const Wrapper = styled("div", {
+  height: "100%",
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+
```

Comment:
?

Reply:
It's the same as having an empty space between some logical parts of code, I guess :)